Tasks/PublishTestResults/publishtestresults.ts

```diff
@@ -1,5 +1,7 @@
 import * as tl from 'vsts-task-lib/task';
 
+const MERGE_THRESHOLD = 100;
```

Comment:
add this to task.json info -> publishTestResults option

"If it exceeds these many number of files, we will auto-merge the results

Reply:
we should ideally write one service to get all test related configuration and consume it here. approving it for now. but to do create WIT